### DISCUSSION TITLE
Labels on Issues

### DISCUSSION THREAD

**Discussion by mickaelGuepe** [renovatebot/renovate#25099] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

Hello,

I would like to add labels on Issues created by Renovate. 
Is it possible to create Issues (Dependency Dashboards for example) with labels ? If not, why it is not implemented as for MR ?

Thanks

### What you tried so far.

I can add labels only on Merge Requests.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
https://docs.renovatebot.com/configuration-options/#dependencydashboardlabels exists

Renovate does not add labels to config error issues though

**Reply by mickaelGuepe** [reply]
Thanks, I didn't find it ... sorry.
Thanks a lot !
